includes/components/class-admin.php

```diff
@@ -946,6 +949,261 @@ protected function get_extensions_tab() {
+						],
+						[
+							'label' => esc_html__( 'External object cache:', 'hivepress' ),
+							'value' => wp_using_ext_object_cache() ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+					],
+				],
+				[
+					'label'  => esc_html__( 'Server environment', 'hivepress' ),
+					'values' => [
+						[
+							'label' => esc_html__( 'Server info:', 'hivepress' ),
+							'value' => isset( $_SERVER['SERVER_SOFTWARE'] ) ? wc_clean( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'PHP version:', 'hivepress' ),
+							'value' => phpversion(),
+						],
+						[
+							'label' => esc_html__( 'PHP post max size:', 'hivepress' ),
+							'value' => ini_get( 'post_max_size' ),
+						],
+						[
+							'label' => esc_html__( 'PHP time limit:', 'hivepress' ),
+							'value' => (int) ini_get( 'max_execution_time' ),
+						],
+						[
+							'label' => esc_html__( 'PHP max input vars:', 'hivepress' ),
+							'value' => (int) ini_get( 'max_input_vars' ),
+						],
+						[
+							'label' => esc_html__( 'cURL version:', 'hivepress' ),
+							'value' => $curl_version,
+						],
+						[
+							'label' => esc_html__( 'SUHOSIN installed:', 'hivepress' ),
+							'value' => extension_loaded( 'suhosin' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'MySQL version:', 'hivepress' ),
+							'value' => $wpdb->db_server_info(),
+						],
+						[
+							'label' => esc_html__( 'Max upload size:', 'hivepress' ),
+							'value' => size_format( wp_max_upload_size() ),
+						],
+						[
+							'label' => esc_html__( 'Default timezone:', 'hivepress' ),
+							'value' => date_default_timezone_get(),
+						],
+						[
+							'label' => esc_html__( 'fsockopen/cURL:', 'hivepress' ),
+							'value' => function_exists( 'fsockopen' ) || function_exists( 'curl_init' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'SoapClient:', 'hivepress' ),
+							'value' => class_exists( 'SoapClient' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'DOMDocument:', 'hivepress' ),
+							'value' => class_exists( 'DOMDocument' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'GZip:', 'hivepress' ),
+							'value' => is_callable( 'gzopen' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'Multibyte string:', 'hivepress' ),
+							'value' => extension_loaded( 'mbstring' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							// todo: check the possibility to test remote post.
+							'label' => esc_html__( 'Remote post:', 'hivepress' ),
```

Comment:
For testing remote post and get, you can use a simple request to wordpress.org website